aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1601,6 +1601,154 @@ Tensor& linalg_norm_out(Tensor& result, const Tensor& self, std::string ord, opt
   return linalg_norm_out_impl(result, self, c10::nullopt, ord, opt_dim, keepdim, opt_dtype);
 }
 
+Tensor _linalg_cond_exception_helper(const Tensor& self) {
+  // For batched input if at least one matrix in the batch is not invertible,
+  // we can't get the result for all other (possibly) invertible matrices in the batch without an explicit for loop.
+  // This should change when at::inverse works with silent errors
+  if (self.dim() > 2) {
+    TORCH_CHECK(false,
+      "At least one matrix in the batch is not invertible, its condition number is infinity, "
```

Comment:
Can we tell which matrix or which matrices aren't invertible? If so, we should include the first non-invertible matrix in this error message and the total number of non-invertible matrices.

If we can't tell, I suggest simplify this to:

"One or more matrices in the batch was not invertible!"

Reply:
No, we can't tell how many matrices are non-invertible and it's not easy to extract the index of the first non-invertible one either.
I'll use your suggestion.